integration_test/gce/gce_testing.go

```diff
@@ -2068,6 +2074,10 @@ func RunForEachImage(t *testing.T, testBody func(t *testing.T, imageSpec string)
 	imageSpecs := strings.Split(imageSpecsEnv, ",")
 	for _, imageSpec := range imageSpecs {
 		imageSpec := imageSpec // https://golang.org/doc/faq#closures_and_goroutines
+		// FIXME(b/398862433): Re-enable tests when writing windows implementation
+		if IsOpsAgentUAPPlugin() && strings.Contains(imageSpec, "windows")  {
```

Comment:
nit: Also use `gce.IsWindows` instead of `strings.Contains(imageSpec, "windows")`.